Makefile

```diff
@@ -1,10 +1,21 @@
 SRC_FOLDER	= src/
+OBJ_FOLDER	= ofiles/
 
+NAME		= minishell
+
+CFILES		:=
+# vpath adds a path to find files(%.c)
+vpath %.c $(SRC_FOLDER)
+CFILES += $(NAME).c
 
-CFILES		= \
-			$(SRC_FOLDER)$(NAME).c\
+vpath %.c $(SRC_FOLDER)exe
+CFILES += exe.c
+CFILES += exe_bin.c
+CFILES += exe_utils.c
 
-OFILES 		= $(CFILES:.c=.o)
+#notdir removes all path from filename 
+#addprefix adds something to each file, in this case the output folder
+OFILES      = $(addprefix $(OBJ_FOLDER), $(notdir $(CFILES:.c=.o)))
```

Comment:
@jhelbig42 I would like to change the structure of the Makefile by introducing the vpath macro. If you agree, we have to be careful when merging this, to use the same synthax for both our feature branches